### PR TITLE
Integer class complete, with unit tests

### PR DESCRIPTION
**Pull request by ryan-b-writin**
Hooray!
@katerebekah

### CODE REVIEW

File: Mathmagician.Tests/IntegerTests.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            int next_last_int = int_sequence.integers[3];
+            Assert.IsTrue(last_int != next_last_int);
+        }
+        [TestMethod]
+        public void WhatHappensIfYouInputZero()
+        {
+            Integer int_with_zero = new Integer();
+            int_with_zero.GenerateIntegers(0);
+            Assert.IsTrue(int_with_zero.integers.Count == 0);
+        }
```
Comment (katerebekah) [inline Mathmagician.Tests/IntegerTests.cs:41]: You need to make a test that just tests that it returns the same number you expect it to, like 1, 2, 3, or 17 or whatever.

File: Mathmagician/Integer.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class Integer
+    {
+        public int newInteger = -1;
+        public List<int> integers = new List<int>();
+        public void GenerateIntegers(int numOfIntegers)
```
Comment (katerebekah) [inline Mathmagician/Integer.cs]: If you are unit testing this method, you should unit test that the list is being added to as expected. Test all the functionality this outputs

**Comment by Odigene12** [comment]
I do not understand what is exactly going on in either tests for the integer tests

**Comment by katerebekah** [comment]
@Odigene12  What don't you understand about the tests? Maybe @ryan-b-writin  could comment them to make them more readable by someone who didn't write the test/code? If your lack of understanding is based on unit testing in general, then that's a different story. You can explore resources like [MSDN](https://msdn.microsoft.com/en-us/library/hh694602.aspx) and [Pluralsight](https://www.pluralsight.com/courses/mstest) in addition to asking questions of the instructors. We are all happy to help!

**Comment by ryan-b-writin** [comment]
@Odigene12 
@jcockhren 
@katerebekah 
@BlaiseGratton

**Comment by ryan-b-writin** [comment]
This class now outputs an array instead of writing to the console from the class
@Odigene12 @jcockhren @katerebekah @BlaiseGratton

**Comment by jcockhren** [comment]
:+1:

**Comment by Odigene12** [comment]
:+1:

**Pull request merged**